### ISSUE TITLE
A11y_SQL Azure Data Studio_Create new server group_Keyboard: Keyboard focus is not visible on new Server Group (MADE BY USER) in the left menu

### ISSUE THREAD

**Issue by Yash14j** [microsoft/azuredatastudio#12959]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) – Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudio_Oct2020; #SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;# WCAG2.4.7;#Benchmark;#keyboard;#FocusVisible;#AINotInScope;

### **Environment Details:**
Application Name: SQL Azure Data Studio
Version 1.22.1 User Setup 
Windows Version: Windows10

### **Repro Steps:**

1.	Launch Azure Data Studio.
2.	Navigate to connection button with the help of tab key and press enter to select.
3.	Tab to 'Servers' header and with the help of tab key navigate to 'New Server Group' icon and press enter.
4.	‘Add Server Group’ screen gets open
5.	Enter the name for the Server group name and a description. Select a colour for the group from the provided colour 
        options. Navigate to 'OK’ button with the help of tab key and press enter
6.	In left menu the new Server Group (MADE BY USER) has been successfully connected.
7.	With the help of Tab key navigate 'Servers' header
8.	Now with the help of Down arrow key navigate to that new server group and check whether focus is visible or not on it

### **Actual:**
Keyboard focus is not visible on new Server Group (MADE BY USER) in the left menu when user navigates to it with the help of Down arrow key.
### **Expected:**
When user navigates to new Server Group in left menu with the Down arrow key, keyboard focus should be visible to user on that new Server Group.
### **User Impact:**
Keyboard user will face difficulty in navigating if keyboard focus is not visible to user.
### **Attachment:**
[#12959_A11y_ SQL Azure Data Studio _Create new server group_Keyboard.zip](https://github.com/microsoft/azuredatastudio/files/5390735/12959_A11y_.SQL.Azure.Data.Studio._Create.new.server.group_Keyboard.zip)
